app/views/account/admin/user/_part/_sub_menu.html.haml

```diff
@@ -0,0 +1,5 @@
+-if yield(:subpage)=="users"
+  %h6(class="mr-5 mt-3 d-inline")
+    =link_to "USERS(#{@users ? @users.size : 0})", account_admin_root_path, :class => "text-danger"
```

Comment:
users will be set to an empty collection if nothing is found 
no need in ternary